script/output_data/output_data.go

```diff
@@ -28,7 +28,7 @@ func generate() error {
 
 	for path, mapping := range beater.Routes {
 
-		if path == beater.HealthCheckURL {
+		if path == beater.HealthCheckURL || path == beater.SourcemapsURL {
```

Comment:
Is there still a reason for skipping sourcemaps here?

Reply:
What do you think about adding sourcemaps to a `docs/generated/sourcemaps` folder, and not to the `intake-api` ? So we would have it documented but clearly distinguished from standard intake API.
I suggest to merge this and add the doc script in a seperate PR.

Reply:
no, updated